week_1/project/week_1.py

```diff
@@ -50,16 +50,25 @@ def get_s3_data(context):
     return output
 
 
-@op
-def process_data():
-    pass
+@op(
+    ins={'stocks': In(dagster_type=List[Stock])},
+    out={"aggregation": Out(dagster_type=Aggregation)},
+    description="Receives list from S3 Data and selects highest stock value"
+)
+def process_data(stocks: List[Stock]) -> Aggregation:
+    stock_high_list = [stock.high for stock in stocks]
+    highest_stock = heapq.nlargest(1,stock_high_list)
```

Comment:
I wasn't familiar with `heapq` so that was nice to see how you used it!

```suggestion
    highest_stock = float(heapq.nlargest(1,stock_high_list)[0])
```
However, I wasn't able to get the tests to pass. I think it's because `heapq.nlargest()` returns a list and `stock_high_list.index()` is expecting a single value. Using the above I was able to get it to work.